### ISSUE TITLE
Create Anaconda distribution

### ISSUE THREAD

**Issue by ryanbaumann** [mapbox/mapboxgl-jupyter#65]
We should create an auto-deployed `Anaconda` distribution of `mapboxgl-jupyter`.  Data scientists and analysts frequently use the Anaconda distribution of Python.  

Process: https://conda.io/docs/user-guide/tutorials/build-pkgs-skeleton.html

**Comment by ryanbaumann** [comment]
Inital conda version of mapboxgl-jupyter is published at:

`conda install -c rsbaumann mapboxgl`


### To Do
- [ ] Check distribution on python 2_7 and 3_5+, and on macOS vs. Windows vs. Linux
- [ ] Add docs to deploy process, or webhook to autodeploy 
- [ ] Use mapbox conda organization cc @sgillies @perrygeo

**Comment by danielfrg** [comment]
I would recommend publishing this to [conda-forge](https://github.com/conda-forge). Its a set of community maintained recipes that will also build the conda packages for win, osx, and linux and publish it to the conda-forge channels where some anaconda users get packages.

The process is relatively simple: take the conda recipe and make a PR.
You can read more about here: https://github.com/conda-forge/staged-recipes

**Comment by ryanbaumann** [comment]
Excellent, thanks for the tip @danielfrg.  `conda-forge` it is.

**Comment by sgillies** [comment]
Yeah, 👍 on conda-forge. I know some of the principals and they've won me over, at least for getting Rasterio into the hands of desktop and notebook users.

**Comment by DruidSmith** [comment]
@ryanbaumann Short term, I'm not finding it in your rsbaumann channel via conda

```
!conda install -c rsbaumann mapboxgl -y

Fetching package metadata .................

PackageNotFoundError: Packages missing in current channels:
            
  - mapboxgl

We have searched for the packages in the following channels:
            
  - https://conda.anaconda.org/rsbaumann/win-64
  - https://conda.anaconda.org/rsbaumann/noarch
  - https://conda.anaconda.org/esri/win-64
  - https://conda.anaconda.org/esri/noarch
  - https://repo.continuum.io/pkgs/main/win-64
  - https://repo.continuum.io/pkgs/main/noarch
  - https://repo.continuum.io/pkgs/free/win-64
  - https://repo.continuum.io/pkgs/free/noarch
  - https://repo.continuum.io/pkgs/r/win-64
  - https://repo.continuum.io/pkgs/r/noarch
  - https://repo.continuum.io/pkgs/pro/win-64
  - https://repo.continuum.io/pkgs/pro/noarch
  - https://repo.continuum.io/pkgs/msys2/win-64
  - https://repo.continuum.io/pkgs/msys2/noarch
```

**Comment by ryanbaumann** [comment]
I'll get a version out on Conda-Forge with the latest updates and drop in the readme (and this ticket) ASAP @DruidSmith.  In the meantime you can start with `pip install mapboxgl`

**Comment by rutgerhofste** [comment]
+1 for conda forge

**Comment by Jeeva-Ganesan** [comment]
Please upload for ubuntu linux 64 bit version as well. 

    conda install -c rsbaumann mapboxgl
    Solving environment: failed

    PackagesNotFoundError: The following packages are not available from current channels:

    - mapboxgl

    Current channels:

      - https://conda.anaconda.org/rsbaumann/linux-64
      - https://conda.anaconda.org/rsbaumann/noarch
      - https://repo.anaconda.com/pkgs/main/linux-64
      - https://repo.anaconda.com/pkgs/main/noarch
      - https://repo.anaconda.com/pkgs/free/linux-64
      - https://repo.anaconda.com/pkgs/free/noarch
      - https://repo.anaconda.com/pkgs/r/linux-64
      - https://repo.anaconda.com/pkgs/r/noarch
      - https://repo.anaconda.com/pkgs/pro/linux-64
      - https://repo.anaconda.com/pkgs/pro/noarch

     To search for alternate channels that may provide the conda package you're
    looking for, navigate to

    https://anaconda.org

    and use the search bar at the top of the page.

Thanks.

**Comment by wgova** [comment]
> I didn't upload a win-64 version yet. I'll get a version out on Conda-Forge with the latest updates (v0.6.0 releasing by Sunday) and drop in the readme for this repo ASAP @DruidSmith.
> 
> In the meantime, you can install with `pip install mapboxgl`.

Has this been resolved?

**Comment by Sieboldianus** [comment]
Looks like: I can install it with `conda install -c conda-forge mapboxgl `

see [https://anaconda.org/conda-forge/mapboxgl](https://anaconda.org/conda-forge/mapboxgl)
